quake-terminal@diegodario88.github.io/prefs.js

```diff
@@ -82,7 +82,7 @@ export default class QuakeTerminalPreferences extends ExtensionPreferences {
 		// App ID
 		const rowId = new Adw.ActionRow({
 			title: _("Terminal App ID"),
-			subtitle: "/usr/share/applications/",
+			subtitle: "/usr/share/applications/ or \n /var/lib/flatpak/exports/share/applications/",
 		});
 		generalSettingsGroup.add(rowId);
 
```

Comment:
Perhaps we could opt for a more flexible and in line with the gnome `about` style approach, like the one shown in the screenshot below. What do you think?

![image](https://github.com/diegodario88/quake-terminal/assets/25825145/c7f76d94-662e-4e74-a0da-0a65b9c14c09)

Reply:
I'll push a branch with this initial work. If you're interested in making improvements, feel free to join in! 😊

Reply:
Check out this [link](https://developer.gnome.org/documentation/tutorials/application-id.html) – it could be really helpful in making the `help dialog` as informative as possible!

Reply:
Seems like a good idea. I'll try to help

Reply:
Hi @petre2dor, wanted to give you a heads up that I'm closing this in favor of this [PR](https://github.com/diegodario88/quake-terminal/pull/22). If you have a moment, it would be awesome if you could test the main branch. Thanks!